akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/StreamTcpDocTest.java

```diff
@@ -49,27 +50,25 @@ public static void tearDown() {
   }
 
   final FlowMaterializer mat = ActorFlowMaterializer.create(system);
-  
-  final InetSocketAddress localhost = new InetSocketAddress("127.0.0.1", 8889);
-  
+
   private final ConcurrentLinkedQueue<String> input = new ConcurrentLinkedQueue<String>();
   {
     input.add("Hello world");
     input.add("What a lovely day");
   }
-  
+
   private String readLine(String prompt) {
     String s = input.poll();
     return (s == null ? "q": s);
   }
-  
+
   @Test
   public void demonstrateSimpleServerConnection() {
     //#echo-server-simple-bind
-    final InetSocketAddress localhost = 
-        new InetSocketAddress("127.0.0.1", 8889);
+    final InetSocketAddress localhost =
+      SocketUtils.temporaryServerAddress();
```

Comment:
This will be included in the docs, and the user doesn't know what SocketUtils is.
Perhaps fake this somehow so the included code is still `new InetSocketAddress("127.0.0.1", 8889)`, but we run with a temp address.